help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,178 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Documentation self-service template for the UI
```

Comment:
same comment as for the other page, please remove "self-service template for the UI" from the title metadata